lib/bezier.js

```diff
@@ -795,6 +795,12 @@
           np3 = this.get(e);
 
           arc = utils.getccenter(np1, np2, np3);
+          
+          //also save the t values
+          arc.t_s = s;
```

Comment:
The start (s) and end (e) t-values are most useful. Perhaps the middle (m) is not quite needed.